Blog/view/blogposts.php

```diff
@@ -1,10 +1,13 @@
-<html>
+<?php
+include('Blog/include/functions.php');
+$print = printHeader();
```

Comment:
You're saving this value to `$print` but from what I can tell, you aren't doing anything with `$print` later on in the code. Is this serving a purpose?